### PR TITLE
docs(changelog): prepare next release changelog

### PR DESCRIPTION
**Pull request by robertohuertasm**
If you are ok, we're releasing this today!

### CODE REVIEW

**Comment by JimiC** [comment]
Please await for the translations fixes.

**Comment by JimiC** [comment]
Let's update this and proceed to release.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # Change Log
 
+## 8.4.0 (April 7, 2019)
```
Comment (JimiC) [inline CHANGELOG.md]: The version should be `8.5.0`

Comment (robertohuertasm) [reply CHANGELOG.md]: done!

**Pull request merged**